indico/web/client/js/react/components/FileSubmission.jsx

```diff
@@ -0,0 +1,138 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React, {useCallback, useRef, useState} from 'react';
+import PropTypes from 'prop-types';
+import Dropzone from 'react-dropzone';
+import {Button, Card, Divider, Grid, Header, Icon, Segment} from 'semantic-ui-react';
+
+import {TooltipIfTruncated} from 'indico/react/components';
+import {Param, Translate} from 'indico/react/i18n';
+
+import './FileSubmission.module.scss';
+
+function humanReadableBytes(bytes) {
```

Comment:
What about moving this to util?

Reply:
Hm, I wasn't sure about this. This is really specific to this component and I don't think we will use it anywhere else.